libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/OAuthWebviewHelper.java

```diff
@@ -459,12 +455,12 @@ public void onPageFinished(WebView view, String url) {
         	super.onPageFinished(view, url);
 		}
 
-		@Override
-        public boolean shouldOverrideUrlLoading(WebView view, String url) {
-			boolean isDone = url.replace("///", "/").toLowerCase(Locale.US).startsWith(loginOptions.getOauthCallbackUrl().replace("///", "/").toLowerCase(Locale.US));
+        @Override
+        public boolean shouldOverrideUrlLoading(WebView view, WebResourceRequest request) {
```

Comment:
shouldOverrideUrlLoading(WebView view, String url) is deprecated, using shouldOverrideUrlLoading(WebView view, WebResourceRequest request) instead.

Reply:
Here we are inside the webview client which is called during a login through the web view.